Cargo.lock

```diff
@@ -1752,6 +1752,7 @@ name = "prusti-specs"
 version = "0.1.0"
 dependencies = [
  "proc-macro2",
+ "prusti-common",
```

Comment:
`prusti-common` dynamically links to the JVM, while `prusti-specs` does not because it's a dependency of procedural macro crates.

Reply:
hm.. prusti-shared? prusti-lib? prusti-basics?

Reply:
@viperproject/prusti-dev suggestions?

Reply:
I'm looking for examples in other projects. `rustc` has `rustc_ty_utils` and `rustc_fs_utils`, so we could mimic them with a `prusti-util`. The Readme should specify in 1 line that the crate is meant to only contain utilities that don't depend on rustc nor the JVM.

Reply:
Although the best refactoring might be to make the `prusti-common` not depend on the JVM. But that's for the future.